doc/sphinx/source/utils.rst

```diff
@@ -256,6 +263,12 @@ given by the slurm flags ``--mem``, ``--constraint`` or ``--ntasks``. In general
         },
    }
 
+Some recipes can only be run with ``--max_parallel_tasks=1`` for various reasons (memory issues, diagnostic issues, CMIP3 data used).
+These recipes need to be added to the ``ONE_TASK_RECIPES`` list.
+
+Note that the script has been optimized to use standard SLURM settings to run most recipes while minimizing the computational cost of the jobs and tailored runtime settings for resource-intensive recipes.
+It is only necessary to edit this script for recipes that have been added since the last release and cannot be run with the default settings.
```

Comment:
Good question, I agree with the suggestion but I'm not sue where and how to document it. This would be mostly the case for IPCC-like recipes for which one of us (Tech Lead Team) is usually involved in the review process. So we could try to do our best to keep this script updated when working on such PRs. In fact, I would have saved quite some time if I had saved slurm setting requirements for the last few IPCC PRs I worked on.

Reply:
Maybe it could be added as a checkbox on the template, but anyway it's not really relevant to what this PR is changing

Reply:
I would make that a required field in the recipe via Core, maybe a good idea in the future: `required_memory_estimate` or something like that, but that may be a bit too much to ask - dunno, maybe we can talk about it, defo not in this PR